devito/propagator.py

```diff
@@ -265,6 +290,9 @@ def generate_loops(self, loop_body):
         :param loop_body: Statement representing the loop body
         :returns: :class:`cgen.Block` representing the loop
         """
+        # Need to set this pragma before generating loops but after stencil substitution
+        self.set_aligned_pragma(self.sub_stencils, self.factorized, self.loop_counters, self.time_steppers)
```

Comment:
PEP8 standard is 79: https://www.python.org/dev/peps/pep-0008/#maximum-line-length

I think we should stick to lines shorter than 90 and change the flake rule accordingly